.github/workflows/build-master.yml

```diff
@@ -5,7 +5,10 @@ on:
     branches:
       - master
   schedule:
-    - cron: 0 10 * * 1
+    # <minute [0,59]> <hour [0,23]> <day of the month [1,31]> <month of the year [1,12]> <day of the week [0,6]>
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+    # Run every Monday at 18:00:00 UTC (Monday at 10:00:00 PST)
+    - cron: '0 18 * * 1'
```

Comment:
is this building from master every monday?   why?  (Why not on merges into master?)

Reply:
It's doing both! Builds master on every push to master but also on CRON. I like having CRON's because maybe we haven't pushed to master in a while but a dependency released a new version that breaks our package, the CRON will be able to tell us something broke regardless.